### PR TITLE
Feat/add doc types

### PR DESCRIPTION
**Pull request by yuhgto**
Added types and tests in two categories – doc types, and miscellaneous missing types. 

**For docs:**

_Added 2 types:_
- BlockTypes = enum to represent which blocks we support
- BlockContent = object to represent the content of a block

_Added 4 docs execute methods:_
- addDocBlock
- updateDocBlock
- addMultiBlocks
- closeDocModal

**For the miscellaneous methods:**

- Added _monday.execute('valueCreatedForUser')_ to notify monday that the user got the first value
- Added _monday.set_ to update the settings in a board view/widget
- Added _SettableTypes_, an enum for the type of data we support using monday.set

### CODE REVIEW

File: types/client-execute.interface.ts [hunk 1]
```diff
@@ -1,3 +1,13 @@
+/**
+ * Blocks that are supported by our SDK
+ */
+type BlockTypes = 'normal text' | 'large title' | 'medium title' | 'small title' | 'bulleted list' | 'numbered list' | 'quote' | 'check list' | 'code';
```
Comment (gregra81) [inline types/client-execute.interface.ts:4]: 🎖️

File: ts-tests/monday-sdk-js-module.test.ts [hunk 2]
```diff
@@ -26,7 +32,31 @@ monday.execute('triggerFilesUpload', {
...
+    afterBlockId : '1234-1234-23434dsf',
+    blocks: [
+        { 
+            type: 'normal text', 
+            content: { deltaFormat : [{ 'insert' : 'test' }] }
+        }
+    ]
+});
+monday.execute('closeDocModal'); // $ExpectType Promise<any>
+
```
Comment (gregra81) [inline ts-tests/monday-sdk-js-module.test.ts:57]: 👏

File: tsconfig.json [hunk 3]
```diff
@@ -14,7 +14,9 @@
     },
     "files": [
         "types/index.d.ts",
-        "ts-tests/monday-sdk-js-module.test.ts",
-        "ts-tests/monday-sdk-js-global.test.ts"
+        "types/client-api.interface.ts",
+        "types/client-data.interface.ts",
+        "types/client-execute.interface.ts",
```
Comment (gregra81) [inline tsconfig.json:19]: these are already imported by `types/index.d.ts` so no need to add them

**Review by gregra81 — APPROVED** [review]
Great job 💯 
See one comment I left you about tsconfig

File: tsconfig.json (hunk 3 above)
Comment (yuhgto) [reply tsconfig.json]: Done – can be merged

**Pull request merged**